src/storage.ts

```diff
@@ -1,12 +1,13 @@
-import { type ChartState } from './types'
+import { GenerateDefaultSiteOptions, GlobalSiteOptions } from './shared'
+import { type SiteOptions, type ChartState } from './types'
 
 // // // //
 // Other
 // // // //
 export function getAllSavedKeys(): string[] {
 	// Needs to be more strictly filtered. Maybe some kind of key for the App submitted entries.
 	return Object.keys(localStorage).filter(
-		(current) => current !== 'CurrentChart'
+		(current) => current !== 'GlobalSiteOptions'
```

Comment:
Since Name collision was mentioned as a worry, it would be better to change how we are storing this name, rather than prevent the user from storing it. Hash would probably work?

Reply:
I think it's not a big deal to have one string of 'GlobalSiteOptions' being restricted. i think dealing with hash and everything is in this case over engineering given any future site things that need to be saved outside of a current chart can just be slapped in there.